### PR TITLE
changed name of method to adhere to the documentation of the module

### PR DESCRIPTION
**Pull request by EreboPSilva**
# Requirements
When creating your Pull request, please fill out the template below:

# PR details
fix

method name for Slice was old/deprecated?

no JIRA tickets

# Testing
I have tested it on the new enviroment, not in the old one but @swatiebi reports that this was a known fix independently made

# Assign to the weekly GitHub reviewer
@ens-ftricomi @leannehaggerty

### CODE REVIEW

File: modules/Bio/EnsEMBL/Analysis/Hive/RunnableDB/CleanUTRs.pm [hunk 1]
```diff
@@ -148,8 +148,9 @@ sub pre_cleanup {
   if ($self->param_is_defined('target_db')) {
     my $db = $self->get_database_by_name('target_db');
     my $slice = $db->get_SliceAdaptor->fetch_by_name($self->input_id);
+	print $self->input_id;
```
Comment (ens-ftricomi) [inline modules/Bio/EnsEMBL/Analysis/Hive/RunnableDB/CleanUTRs.pm:151]: could you print remove the print?

**Review by ens-ftricomi — CHANGES_REQUESTED** [review]
it looks fine to me

File: modules/Bio/EnsEMBL/Analysis/Hive/RunnableDB/CleanUTRs.pm (hunk 1 above)
Comment (EreboPSilva) [reply modules/Bio/EnsEMBL/Analysis/Hive/RunnableDB/CleanUTRs.pm]: done!

**Review by ens-ftricomi — APPROVED** [review]
thank you

**Pull request merged**